pallets/order-book/src/lib.rs

```diff
@@ -120,6 +120,14 @@ pub mod pallet {
 		/// Fee Key used to find amount for allowance reserve/unreserve
 		type OrderFeeKey: Get<<Self::Fees as Fees>::FeeKey>;
 
+		/// Token Id for token used for fee reserving
+		/// as represented by `AssetCurrencyId`.
+		/// Used for reserve-able fund checking to ensure
+		/// amount traded and storage fees can be reserved
+		/// when trading for fee currency.
+		/// This should typically be native chain currency.
+		type FeeCurrencyId: Get<Self::AssetCurrencyId>;
```

Comment:
When this will not be the native currency? Is it just a way to identify our currency in the `MultiReservableCurrency`?

Reply:
Yeah exactly :) -- there aren't any forseeable situations where it wouldn't be native, but we still need to be able to see if it is native based off the asset id.